src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -187,6 +197,9 @@ public function loadClass($class)
                     if ($parent && isset(self::$deprecated[$parent]) && strncmp($ns, $parent, $len)) {
                         @trigger_error(sprintf('The %s class extends %s that is deprecated %s', $name, $parent, self::$deprecated[$parent]), E_USER_DEPRECATED);
                     }
+                    if ($parent && isset(self::$final[$parent]) && strncmp($ns, $parent, $len)) {
+                        @trigger_error(sprintf('Extending the class %s in %s is deprecated and may break in the next major release.', $parent, $name), E_USER_DEPRECATED);
```

Comment:
you don't add a deprecation, you add a ``@final`` tag, which does not have a reason to include a version IMO.
and btw, the next major version could decide to keep the class as virtually final instead of making the final for real, if they want to keep them mockable for instance.